### DISCUSSION TITLE
Run locally on windows

### DISCUSSION THREAD

**Discussion by Secrios** [JoePenna/Dreambooth-Stable-Diffusion#81] [Ideas]
I have took a dive into having this run locally with jupyter labs on my windows 10, but I now realised half the code here is geared for linux.

Could you make a special .ipynb for local windows usage?

I currently have python & git on path. and java generically installed. I also use an RTX 3090.

**Comment by chakalakasp** [comment]
Not the repo owner, but yeah, that'd be cool!  Thought about spinning up a linux VM to get around this but the more I look into it the more I realize what a crazy headache that would be, as passing through the raw GPU is not something that's easy or convenient to do.

**Reply by smc-git** [reply]
yeah got 4090 would love to have this run local have dual gpus so can have enuough free to dedicate
would beat having to boot to linux for training
